src/modules/search-time/selector/index.tsx

```diff
@@ -83,8 +84,8 @@ export default function SearchTimeSelector({
       resetToCurrentDate();
 
     // To ensure that the time is not past whenever reselecting back the current date.
-    if (selectedDate <= today) resetToCurrentTime();
-    return selectedDate < today;
+    if (selectedDate <= yesterday) resetToCurrentTime();
+    return selectedDate < yesterday;
   };
```

Comment:
I guess this kind of belongs the original PR for this function, but this function seems a bit odd to me. It returns a boolean for whether the date is in the past or not, but also calls the `resetToCurrentTime`. I think it would make more sense if this function returns whether the date is in the past, or with this change, if it is past yesterday, and if so the calling function calls `resetToCurrentTime`.